app/config/config.yml

```diff
@@ -11,6 +11,13 @@ imports:
 
 # Basic configuration for the Symfony framework features
 framework:
+    # Uncomment the 'ide' option to turn file paths into clickable links that
+    # open the given file path and line number using your favorite IDE.
+    # Supported values are 'textmate', 'macvim', 'emacs' and 'sublime'
+    # See http://symfony.com/doc/current/reference/configuration/framework.html#ide
+    #
+    # ide: sublime
```

Comment:
@aitboudad it is useful for any environment you could run locally. Even though you generally run the dev env locally, it is not the only one. I regularly switch to app.php when I want to perform some profiling for instance, to avoid the overhead of all the symfony debugging tools, which is not relevant for the performance of my app.